components/compliance-service/ingest/projectupdater/manager.go

```diff
@@ -177,12 +181,34 @@ func (manager *Manager) waitingForJobToComplete() {
 		}
 	}
 
-	logrus.Debugf("Finished Project rule update with Elasticsearch job ID: %q and projectUpdate ID %q",
-		manager.esJobID, manager.projectUpdateID)
+	logrus.Debugf("Finished Project rule update")
 
 	manager.state = notRunningState
 }
 
+func (manager *Manager) combinedJobStatus() (ingestic.JobStatus, error) {
+	combinedJobStatus := ingestic.JobStatus{
+		Completed:          true,
+		PercentageComplete: 1.0,
+	}
+	for _, esJobID := range manager.esJobIDs {
+		jobStatus, err := manager.client.JobStatus(context.Background(), esJobID)
+		if err != nil {
+			return jobStatus, err
+		}
+
+		if !jobStatus.Completed {
+			combinedJobStatus.Completed = false
```

Comment:
This feels wrong to me. Doesn't this mean the first job in the array could be still going and the second one is completed, then the return value would be that both of them are completed?

Reply:
I will add some tests to this function

Reply:
@phiggins here is a test for that case now https://github.com/chef/automate/pull/86/files#diff-5bef6c2a36a3b9fb536231672b4dc731R82